src/current/_includes/v23.1/sidebar-data/cloud-deployments.json

```diff
@@ -95,7 +95,54 @@
             ]
             },
             {
-            "title": "CockroachDB Dedicated on Azure",
+            "title": "Migrate from Serverless to Dedicated",
```

Comment:
Nit: Update the title? A few more instances below.

Reply:
Nope -- only updated the sidebar entries for docs updated in this PR